test/deploy/linux/kafka/install/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- debug:
+    msg: Starting Zookeper and Kafka Server
+
+- block:
+    - name: Start Zookeeper
+      shell: cd /opt/kafka/ && sudo bin/zookeeper-server-start.sh config/zookeeper.properties &
+      become: true
+      async: 5
```

Comment:
@NRhzhao why do we have to wait up to 5s for this task to complete? Could we just run it once and wait for its completion?

Reply:
@Julien4218 the zookeeper/kafka script would block the console because it doesn't 'complete'.  So, the ansible job never really finishes.  With Linux, I think it's okay to run the process in the background with &, but for Windows, I have to manually timeout so it would move to the 2nd part of the script.  Not sure if there is a better way of handling it.

I keep the timeout just to keep consistentcy.

Reply:
Ok, so if this is not needed for linux, let's not have it, I have a separate comment for windows to not rely on poll/async